lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -212,29 +207,10 @@ def mock_run(args, **kwargs):
                     args, returncode=0, stdout="Successfully Unpacked!", stderr=None
                 )
             else:
-                assert args == [
-                    "find",
-                    ".",
-                    "(",
-                    "-type",
-                    "d",
-                    "-exec",
-                    "chmod",
-                    "ugo+rx",
-                    "'{}'",
-                    "+",
-                    ")",
-                    "-o",
-                    "(",
-                    "-type",
-                    "f",
-                    "-exec",
-                    "chmod",
-                    "ugo+r",
-                    "'{}'",
-                    "+",
-                    ")",
-                ]
+                assert args[0] == "find"
+                assert kwargs["cwd"] == (
+                    server_config.INCOMING / "ABC" / f"{tarball_name}.unpack"
+                )
```

Comment:
Are you sure you want to duplicate this assertion from line 201, rather than moving it to common code (e.g., before line 197)?  (Alternately, you could modify the function to have a common return point and place the assertion after the conditional before the `return` statement.)